mautic/mautic
# Fix set values from properties for state field

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR fixes a lot of notices in logs:

`mautic.NOTICE: PHP Notice - Undefined index: value - in file...`

This coming from form's send form result action If you trying send state field.

Also it fix

`PHP Notice - Array to string conversion `

when you open that state field to editation



#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. Create from with email and state  - state match with lead field state and enable
![image](https://user-images.githubusercontent.com/462477/135098618-ef818f58-01ec-470c-9b97-6fabd4d897cd.png)
3. Add action send form results to email
4. Try submit to form 
5. You should see notice in logs: `mautic.NOTICE: PHP Notice - Undefined index: value - in file...`
6. If you re-open edit form, then try edit state field, you should see in logs error `PHP Notice - Array to string conversion`
6. After PR both should works
